mix.exs

```diff
@@ -19,16 +19,15 @@ defmodule Ueberauth.Mixfile do
   end
 
   def application do
-    [applications: [:logger]]
+    [extra_applications: [:logger]]
```

Comment:
If I am not mistaking, by doing this, we officially support only `1.4+` ? Or whichever changed the `application` config, I forgot already 😅

Reply:
I think that is right - I chnaged it when I noticed that the elixir version in the `mix.exs` file is specified as `elixir: "~> 1.4"` .. I wonder is it already only supporting 1.4 + or is that a discretionary keyword.. not sure. I can change this back if needed?